app/schema_names.rb

```diff
@@ -0,0 +1,7 @@
+class SchemaNames
+  # Return the names of all schemas, except `content_block_email_address` which is
+  # still in development
```

Comment:
I suggest adding a link to the upstream PR, alongside a "TODO" comment along the lines of "remove this when the PR is merged".